zio-http/src/main/scala/zio/http/RoutePattern.scala

```diff
@@ -64,6 +64,13 @@ final case class RoutePattern[A](method: Method, pathCodec: PathCodec[A]) { self
   ): Route[Env, Err] =
     Route.route(Route.Builder(self, HandlerAspect.identity))(handler)(zippable.zippable, trace)
 
+  def -->[Env, Err, I, Ctx, In](handler: Handler[Env, Err, I, Response])(implicit
```

Comment:
If we are going to use `-->` for this, then we should make sure to be consistent across all of ZIO HTTP.

In addition: We should give this a named variant too, rather than just the symbolic operator.

Reply:
Can you be more specific on being consistent with what? I don't even say we should use it, but it was not possible to simply overload `->`